compiler/src/iree/compiler/Codegen/WGSL/Passes.td

```diff
@@ -13,11 +13,4 @@ include "mlir/Pass/PassBase.td"
 // WGSL passes (keep alphabetical)
 //===---------------------------------------------------------------------===//
 
-def WGSLReplacePushConstantsPass :
```

Comment:
Can you drop the usage from the plugin too? That should keep the code building.
https://github.com/iree-org/iree/actions/runs/10515973568/job/29137201737?pr=18326#step:4:8662
```
/work/compiler/plugins/target/WebGPUSPIRV/WebGPUSPIRVTarget.cpp:127:9: error: use of undeclared identifier 'createWGSLReplacePushConstantsPass'
        createWGSLReplacePushConstantsPass());
        ^
```

I'm also expecting some test failures. Can see what those are when the other failures on the `shared/dispatch2` branch are resolved, then disable or mark them xfail.

Reply:
oh whoops